src/controllers/import.js

```diff
@@ -105,7 +105,11 @@ function ImportController(context) {
       validateImportApiKey(importApiKey);
       validateRequestData(data);
 
-      const { urls, options } = data;
+      const { urls } = data;
+      let { options } = data;
+      if (!options) {
+        options = importConfiguration.options;
+      }
```

Comment:
Think this can be simplified a bit:
```suggestion
      const { urls, options = importConfiguration.options } = data;
```